doc/help.md

```diff
@@ -889,6 +889,86 @@ JOB:
+followed by a closing ']'.
+
+Tips:
+
+Don't forget to leave a space between the brackets and the expression. "[" is
+a command, not syntax, so the shell will fail to recognize it without the
+spaces.
+
+
+<h4 id="test">test</h4>
+
+    test EXPR+
+
+Evaluates a conditional expression and returns 0 (true) or 1 (false) as result.
+The number of elments in the expression depends on the type of test.
+
+File tests:
+
+    -a FILE  True if FILE exists (recommended to use -e instead).
+    -b FILE  True if FILE is a block special file.
+    -c FILE  True if FILE is a character special file.
+    -d FILE  True if FILE is a directory.
+    -e FILE  True if FILE exists.
+    -f FILE  True if FILE is a regular file.
+    -g FILE  True if FILE has the sgid bit set.
+    -G FILE  True if current user's group is also FILE's group.
+    -h FILE  True if FILE is a symbolic link.
+    -L FILE  True if FILE is a symbolic link.
+    -k FILE  True if FILE has the sticky bit set.
+<!--    -N FILE  True if FILE was modified since last read (mtime newer than atime).-->
+    -O FILE  True if current user is the file owner.
+    -p FILE  True if FILE is a named pipe (FIFO).
+    -r FILE  True if FILE is readable.
+    -s FILE  True if FILE has size bigger than 0.
+    -S FILE  True if FILE is a socket file.
+    -t FD    True if file descriptor FD is open and refers to a terminal.
+    -u FILE  True if FILE has suid bit set.
+    -w FILE  True if FILE is writable.
+    -x FILE  True if FILE is executable.
+    FILE1 -nt FILE2  True if FILE1 is newer than FILE2 (mtime).
+    FILE1 -ot FILE2  True if FILE1 is older than FILE2 (mtime).
+    FILE1 -ef FILE2  True if FILE1 and FILE2 are hard links.
+
+String tests:
+
+    -n STRING           True if STRING is not empty. This is the default
+                        operation if no test is specified.
+    -z STRING           True if STRING is empty.
+    STRING1 = STRING2   True if the strings are equal.
+    STRING1 != STRING2  True if the strings are not equal.
+    STRING1 < STRING2   True if STRING1 sorts before STRING2 lexicographically.
+    STRING1 > STRING2   True if STRING1 sorts after STRING2 lexicographically.
+                        Note: < and > should be escaped, eg: \< and \>
+
+Arithmetic tests:
+
+    INTEGER1 -eq INTEGER2  True if the integers are equal.
+    INTEGER1 -ne INTEGER2  True if the integers are not equal.
+    INTEGER1 -le INTEGER2  True if INTEGER1 is less or equal than INTEGER2.
+    INTEGER1 -ge INTEGER2  True if INTEGER1 is greater than INTEGER2.
+    INTEGER1 -lt INTEGER2  True if INTEGER1 is less than INTEGER2.
+    INTEGER1 -gt INTEGER2  True if INTEGER1 is greater than INTEGER2.
+
+Other operators:
+
+    -o OPTION  True if the shell option OPTION is set.
+    -v VAR     True if the variable VAR has been set.
+<!--    -R VAR     True if the variable VAR has been set and is a nameref variable. -->
+    EXPR1 -a EXPR2  True if expressions EXPR1 and EXPR2 are both true.
```

Comment:
I would be tempted to leave these out as discouraged, and just say it follows POSIX.  But you can leave it alone, I will likely make an editing pass